lib/artsy-eventservice/presenters/events/base_event.rb

```diff
@@ -30,13 +30,18 @@ def object
       end
     end
 
-    def json
+    def to_json
```

Comment:
Nice! Yeah, we should probably just override the `to_json` instead of using a custom `json` method.